client/packages/requisitions/src/ResponseRequisition/api/api.ts

```diff
@@ -212,6 +215,23 @@ export const getResponseQueries = (sdk: Sdk, storeId: string) => ({
 
     throw new Error('Unable to update requisition');
   },
+  deleteResponses: async (requisitions: ResponseFragment[]) => {
+    const deleteResponseRequisitions = requisitions.map(
+      responseParser.toDelete
+    );
+    const result = await sdk.deleteRequest({
+      storeId,
+      input: { deleteResponseRequisitions },
+    });
+
+    const { batchResponseRequisition } = result || {};
+
+    if (batchResponseRequisition?.deleteResponseRequisitions) {
+      return batchResponseRequisition.deleteResponseRequisitions.length;
```

Comment:
If you just return the deleteResponseRequisitions instead of the length, you can use that to check for errors.